### PR TITLE
pagination component

### PR DESCRIPTION
**Pull request by CiotkaCierpienia**

### CODE REVIEW

File: src/components/Pagination/Pagination.js [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+        sm: 'rounded-sm',
+        md: 'rounded-md',
+        lg: 'rounded-lg',
+        full: 'rounded-full',
+    };
+    const safePage = Math.min(Math.max(page, 1), numOfPages);
+
+    return (
+        <div className={[
+            'bg-white',
```
Comment (trzcina) [inline src/components/Pagination/Pagination.js:83]: Can we remove the default `bg-white` class?
It causes a situation like below, when defaults used
image

Comment (CiotkaCierpienia) [reply src/components/Pagination/Pagination.js:83]: yes, we can

File: src/components/Pagination/Pagination.js [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+    next = 'Next',
+    baseUrl = '/',
+    page = 1,
+    numOfPages = 1 }) => {
+    const sizeClass = {
+        sm: 'w-6 h-6 md:w-8 md:h-8 text-sm',
+        md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-base',
+        lg: 'w-10 h-10 md:w-12 md:h-12 text-md md:text-lg',
+    };
+    const sizeClass2 = {
```
Comment (trzcina) [inline src/components/Pagination/Pagination.js:25]: Can we use another name than ...`2`?  I can't find out what does it mean `2` in this name

File: dist/Pagination/Pagination.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  containerAdditionalClasses = [],
+  prev = 'Previous',
+  next = 'Next',
+  baseUrl = '/',
+  page = 1,
+  numOfPages = 1
+}) => {
+  const sizeClass = {
+    sm: 'w-6 h-6 md:w-8 md:h-8 text-sm',
+    md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-base',
```
Comment (trzcina) [inline dist/Pagination/Pagination.js:24]: to be consistent with button, it should be 'text-md' by default, not `base`
```suggestion
    md: 'w-8 h-8 md:w-10 md:h-10 text-sm md:text-md',
```

File: dist/Pagination/Pagination.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+    lg: 'w-10 h-10 md:w-12 md:h-12 text-md md:text-lg'
+  };
+  const sizeClass2 = {
+    sm: 'w-16 h-6 md:w-20 md:h-8 text-sm',
+    md: 'w-20 h-8 md:w-24 md:h-10 text-sm md:text-base',
```
Comment (trzcina) [inline dist/Pagination/Pagination.js:29]: to be consistent with button, it should be 'text-md' by default, not `base`
```suggestion
    md: 'w-20 h-8 md:w-24 md:h-10 text-sm md:text-md',
```

File: src/components/Pagination/Pagination.js (hunk 2 above)
Comment (CiotkaCierpienia) [reply src/components/Pagination/Pagination.js:25]: we can use sizeClassPrevNext

**Pull request merged**